src/commands/graph/graph.mjs

```diff
@@ -32,7 +34,12 @@ export const createGraphCommand = (program) => {
 
   return program
     .command('graph')
-    .description('(Beta) Control the Netlify Graph functions for the current site')
+    .hook('preAction', () => {
+      warn(
+        'Netlify Graph is deprecated and will be removed from the CLI in a future release at the end of April, 2023. Please migrate off of Netlify Graph beforehand.',
+      )
+    })
+    .description('(deprecated) Control the Netlify Graph functions for the current site')
```

Comment:
Super nit, but I think this casing would be more aligned with the _(Beta)_ prefix we use in other commands.

```suggestion
    .description('(Deprecated) Control the Netlify Graph functions for the current site')
```